### PR TITLE
Updating test app with expandable rows table and InlineAlert

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

RSP

### CODE REVIEW

File: examples/rsp-webpack-4/src/App.js [hunk 1]
```diff
@@ -1,12 +1,30 @@
 import './App.css';
-import {Provider, defaultTheme, Item} from '@adobe/react-spectrum'
+import {Provider, defaultTheme, Item, TagGroup, Cell, Column, Row, TableBody, TableHeader, TableView, Content, Heading} from '@adobe/react-spectrum';
 import Lighting from './Lighting';
 import {useState} from 'react'
 import BodyContent from './BodyContent';
 import {TagGroup} from '@react-spectrum/tag';
```
Comment (LFDanLu) [inline examples/rsp-webpack-4/src/App.js:6]:
```suggestion
```

**Comment by reidbarber** [comment]
Links: https://reactspectrum.blob.core.windows.net/reactspectrum/1e4991215bcbb00b746a7648e837680d1a8c9f92/verdaccio/build/index.html

https://reactspectrum.blob.core.windows.net/reactspectrum/1e4991215bcbb00b746a7648e837680d1a8c9f92/verdaccio/next/index.html

**Review by reidbarber — DISMISSED** [review]
Can be follow-up, but there are a few errors (hydration, aria-labels, SSRPovider) we could handle. Render behavior looks good though.

**Comment by LFDanLu** [comment]
@reidbarber  good catch, I'll see if I can fix those real quick

**Comment by reidbarber** [comment]
I'm good to approve after CI changes are reverted.

**Review by reidbarber — APPROVED** [review]

**Review by devongovett — APPROVED** [review]

**Pull request merged**